extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -110,6 +110,7 @@ export class AmpSidebar extends AMP.BaseElement {
     const toolbarElements =
     Array.prototype.slice
       .call(this.element.querySelectorAll('nav[toolbar]'), 0);
+    toolbarElements.reverse();
```

Comment:
If we do not add the element as the first child then the placeholder solution will not work.

The alternative in that case would to be to add border-top to the body. In that case we would have to maintain the `top:` styling of all our toolbars, as well as getting all of our toolbars to play nicely with one another, or having sidebar manage each toolbar's height to calculate top accordingly. Also, in this case, performing #9983 would be much more difficult